autodepgraph/tests/write_test_graphs.py

```diff
@@ -0,0 +1,57 @@
+import autodepgraph as adg
+import autodepgraph.graph as g
+import autodepgraph.node as n
+import os
```

Comment:
I guess this corresponds to creating the datafiles that are used in the tests. Maybe add some comment as to what this file is used for.